packages/newsroom-signup/src/ApplyToTCR/ApplyToTCRSuccess.tsx

```diff
@@ -0,0 +1,107 @@
+  const pad = (num: number | string) => {
+    return padString(num, 2, "0");
+  };
+  const hours = pad(theDate.getHours());
+  const mins = pad(theDate.getMinutes());
+  const tzOffset = `${pad(theDate.getTimezoneOffset() / 60)}${pad(theDate.getTimezoneOffset() % 60)}`;
+  const dateString = `${theDate.getFullYear()}-${pad(theDate.getMonth() + 1)}-${pad(theDate.getDate())}`;
+  const timeString = `${hours}:${mins}-${tzOffset}`;
+  return `${dateString}T${timeString}`;
+}
+
+interface AddApplicationEndToCalendar {
+  endTime: number;
+  listingDetailURL: string;
+}
+
+const AddApplicationEndToCalendar: React.SFC<AddApplicationEndToCalendar> = props => {
+  const title = "My Newsroom review period has ended for the Civil Registry";
+  const description = "My Newsroom review period has ended for the Civil Registry";
+  const location = props.listingDetailURL;
+  const startTime = getCalendarEventDateTime(props.endTime);
+  const event = {
+    title,
+    description,
+    location,
+    startTime,
+  };
+  return (
+    <StyledAddToCalendar>
+      <AddToCalendar event={event} />
+    </StyledAddToCalendar>
+  );
+};
+
+const ApplyToTCRSuccess: React.SFC<ApplyToTCRSuccessProps> = props => {
+  const { listing } = props;
+  const listingAddress = listing.address;
+  const endTime = listing.data.appExpiry.toNumber();
+  const applicationEndDateTime = getLocalDateTimeStrings(endTime);
+  const registryListingURL = `https://registry.civil.co/listing/${listingAddress}`;
+
+  return (
+    <>
+      <OBSectionHeader>Application submitted!</OBSectionHeader>
+      <OBSectionDescription>
+        Your newsroom application was submitted to the Civil Registry will now be reviewed by the Civil Community over
+        the next {props.applyStageLenDisplay}.
+      </OBSectionDescription>
+
+      <FormSection>
+        <FormRow>
+          <FormRowItem>
+            <p>
+              Your newsroom review period ends on {applicationEndDateTime[0]} at {applicationEndDateTime[1]}.
+            </p>
+
+            <p>Please check your emails during that time to keep tabs on your newsroom's process.</p>
+          </FormRowItem>
+          <FormRowItem align="right">
+            <AddApplicationEndToCalendar endTime={endTime} listingDetailURL={registryListingURL} />
+          </FormRowItem>
+        </FormRow>
+        <FormRow>
+          If a community member challenges your newsroom during the review period, you will get an email with
+          recommendations on next steps.
+        </FormRow>
+      </FormSection>
+
+      <FormSection>
+        <OBSectionDescription>
+          You can also follow the progress of your application on your Newsroom Profile on the Registry.
+        </OBSectionDescription>
+
+        <Button size={buttonSizes.SMALL} href={registryListingURL}>
+          View your Newsroom
+        </Button>
+
+        <p>
+          If you have any questions, you can contact <a href="#">support</a>
```

Comment:
Not sure if we have this in a constant anywhere but I think this can be support@civil.co.